iceoryx_hoofs/include/iceoryx_hoofs/internal/units/duration.hpp

```diff
@@ -38,7 +38,7 @@ enum class TimeSpecReference
 
 class Duration;
 
-inline namespace duration_literals
+namespace duration_literals
```

Comment:
I am strongly against it.

Just assume that we have a unit like length and here we have the literal `m` for meter which is identical for `m` in minutes. Then we have ambiguous symbols.

And yes you are right, the STL actually does it like that and this is in my opinion not a good approach because of the reason stated above. But the stl does not have a unit like meter - not yet - but if they gonna get one I bet the stl will become even more complex to handle the problem above.